thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/util/ThirdEyeUtils.java

```diff
@@ -148,4 +150,21 @@ public static String getAliasFromCollection(String collection) throws ExecutionE
     return result;
   }
 
+  public static String constructCron(String scheduleMinute, String scheduleHour, String repeatEvery) {
```

Comment:
`repeatEvery` should be a TimeUnit.
validation on scheduleMinute and scheduleHour
Add unit test(s)